scipy/optimize/_linprog_highs.py

```diff
@@ -245,6 +242,10 @@ def _linprog_highs(lp, solver, time_limit=None, presolve=True,
             4,
             'HiGHS Status Code 6: HighsModelStatusPOSTSOLVE_ERROR',
         ),
+        MODEL_STATUS_MODEL_EMPTY: (
+            4,
+            'HiGHS Status Code 3: HighsModelStatusMODEL_EMPTY',
+        ),
```

Comment:
Was moving this intentional? Any functional change?

Reply:
Yes, it was intentional to match the reordering of the enumeration.  There is no functional change, just makes it easier to compare when it's in the same order